internal-packages/ai/src/workflows/Workflow.ts

```diff
@@ -111,10 +111,13 @@ export class Workflow<Shape extends IOShape = IOShape> {
 
   public llmClient: LLMClient;
 
+  public error?: string;
```

Comment:
Maybe we should/could rename this? "error" is pretty generic - there will be lots of sub-errors in the different steps.

Maybe something like, "fatalError' or 'finalError'?

Reply:
(I want to get this PR in first though, to use it)

Reply:
Maybe... I'm not sure either. But it's a workflow-level error, and there's only one, so extra prefix seems repetitive.

(e.g. what if it evolves in the direction of `error: { type: "FATAL" | "MINOR", value: string }` or something... hard to predict)